### ISSUE TITLE
Do not disturb issues

### ISSUE THREAD

**Issue by Russellfam** [quarck/CalendarNotification#302]
I have huawei p20pro and when I have do not disturb on the notifications from calendar still come through. I cannot see a reason for this and the priority interruptions only has calls and messages no option for calendar events. Any ideas?

**Comment by quarck** [comment]
I don't think it is actually inside the scope of the app to handle DND. It suppose to be handled by the OS, so unless user has set a DND exception for the app - it should not be capable of "breaking the DND wall".
